mcfly.bash

```diff
@@ -27,8 +34,8 @@ function mcfly_initialize {
   export MCFLY_SESSION_ID
 
   # Find the binary
-  MCFLY_PATH=${MCFLY_PATH:-$(command which mcfly)}
-  if [[ -z $MCFLY_PATH ]]; then
+  MCFLY_PATH=${MCFLY_PATH:-$(builtin type -P mcfly)}
```

Comment:
We ran into an issue in the past with which, see https://github.com/cantino/mcfly/issues/292#issuecomment-1328569552 and https://github.com/cantino/mcfly/pull/309. Does builtin fix this?

Reply:
I do not seem to be able to reproduce the original problem of `./bin/mcfly` in a normal setup.

However, I could reproduce the problem by adding `./bin` in `PATH`. In this case, while `command -v` returns `./bin/mcfly`, `which` (GNU which) seems to return the absolute path `/bin/mcfly`. I also checked the behavior of `builtin type -P mcfly`, which produces `./bin`. To me, this appears to be a user error of including a relative path in `PATH`.

Anyway I can add a workaround in a different way without using non-POSIX `which`.

Reply:
I fixed up the corresponding commit: 0c0e0b52. I also described the background in the commit message.

Reply:
Thanks, and sorry for the delay!